### PR TITLE
[new-release] cborl (0.1.0)

### PR DESCRIPTION
**Pull request by pukkamustard**
A [CBOR](https://www.rfc-editor.org/rfc/rfc8949.html) library for OCaml. Supports streaming decoding and encoding.

Repository: https://inqlab.net/git/ocaml-cborl.git/

### CODE REVIEW

**Comment by pukkamustard** [comment]
I've fixed the 64bit integer literal (https://inqlab.net/git/ocaml-cborl.git/commit/?id=1912134ab8c5767047046d6ce2b3b23834f6c074). The fix will be released in a version 0.2.0.

I'm a bit confused about the lint/parsing error: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/0391d9a79c2539a8199016f2a6bbf1df2f0a4fbc/variant/(lint)

The `url` block looks in-order and has a `checksum` field. I don't understand what is wrong.

File: packages/cborl/cborl.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://inqlab.net/git/ocaml-cborl.git"
+url {
+  src: "https://inqlab.net/projects/ocaml-cborl/release/ocaml-cborl-v0.1.0.tar.gz"
+  checksum: [
+    "md5=855106fd1a3bed94af232eb168a51fd0"
+    "sha512=b065f4fb7961b8f7fbcffe0ee1cd8c7cfdd447b200a81e908fee00e73096a1e5"
```
Comment (mseri) [inline packages/cborl/cborl.0.1.0/opam:44]:
```suggestion
    "sha256=b065f4fb7961b8f7fbcffe0ee1cd8c7cfdd447b200a81e908fee00e73096a1e5"
```
I think the error is due to this.

**Comment by mseri** [comment]
Thanks

File: packages/cborl/cborl.0.1.0/opam (hunk 1 above)
Comment (pukkamustard) [reply packages/cborl/cborl.0.1.0/opam]: Aha! Thanks for finding this!

**Pull request merged**